examples/basic/package.json

```diff
@@ -14,7 +14,7 @@
     "react-dom": "^16.0.0"
   },
   "devDependencies": {
-    "raf": "^3.3.2",
+    "raf": "^3.4.0",
```

Comment:
We should remove raf from all example and include config/polyfilly in jest config like CRA does.